FIX potential detrimental tf api usage

TITAN-PyCompat opened the pull request:
Dear developers,

When I use your project, I found some possible detrimental TensorFlow API use in your code.

As shown in my pull request: `tf.multinomial` has been deprecated since tf v1.13.0 and **removed in tf 2.0.0-alpha**. It's better to replace it with `tf.random.categorical`. Otherwise, crash might happens for users who use the newest version of TensorFlow.
@soloice @burness 
Hope you could take my advice and look forward to your reply! 😃

burness commented:
@TITAN-PyCompat  Have you test the code in your project ? If yes, I will merge it

TITAN-PyCompat commented:
@burness  Thanks for your reply! 
I have tested those code on different versions of TensorFlow and it runs well.

By the way, I added some modifications to another place:

`tf.logging._logger` which is a private API has been removed or changed in some version. And When I run on tf v 1.13.1 it **crashed**.

I changed your code 
```python
_logger = tf.logging._logger
_logger.setLevel(0)
```   
to 
```python
tf.logging.set_verbosity(0)
```
Both of them perform the same things and the modified one is compatible on all versions of TensorFlow.

And then it runs normally!

You could merge these changes now! 
Thanks!

burness commented:
Thanks for your contributions, I will merge it
